src/test/kotlin/blackjack/domain/PlayerStateTest.kt

```diff
@@ -0,0 +1,72 @@
+package blackjack.domain
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertAll
+
+class PlayerStateTest {
+    private val hitOrStayScore = Score.from(
+        PlayingCards.from(
+            listOf(
+                PlayingCard.of(Suit.CLUBS, CardNumber.NINE),
+                PlayingCard.of(Suit.HEARTS, CardNumber.ACE)
+            )
+        )
+    )
+    private val blackjackScore = Score.from(
+        PlayingCards.from(
+            listOf(
+                PlayingCard.of(Suit.CLUBS, CardNumber.KING),
+                PlayingCard.of(Suit.HEARTS, CardNumber.ACE)
+            )
+        )
+    )
+    private val bustScore = Score.from(
+        PlayingCards.from(
+            listOf(
+                PlayingCard.of(Suit.CLUBS, CardNumber.KING),
+                PlayingCard.of(Suit.HEARTS, CardNumber.QUEEN),
+                PlayingCard.of(Suit.HEARTS, CardNumber.JACK)
+            )
+        )
+    )
+
+    private val hitState = PlayerState.of(hitOrStayScore, true)
+    private val stayState = PlayerState.of(hitOrStayScore, false)
+    private val blackjackState = PlayerState.of(blackjackScore)
+    private val bustState = PlayerState.of(bustScore)
+
+    @Test
+    fun `PlayerState는 점수와 진행 의지에 따른 현재 플레이어의 상태를 나타낸다`() {
+        assertAll(
+            { assertThat(hitState).isInstanceOf(PlayerState.Hit::class.java) },
```

Comment:
확장함수를 만들어 표현을 개선해 봅시다.
```suggestion
            { assertThat(hitState).isA<PlayerState.Hit>() },
```